net/l3roamd/Makefile

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=l3roamd
+PKG_VERSION:=1
```

Comment:
This line can be removed. Instead, add a PKG_SOURCE_DATE option set to the date of the commit 9d474a9329e307a5e71fd55d53491d93d34dda13, as done in https://git.lede-project.org/?p=source.git;a=blob;f=package/system/procd/Makefile;h=fd1bca3f4b9665581f9e9ddd9b9294fa500a9d3d;hb=HEAD